docs/Prefill-decoding_Disaggregation.md

```diff
@@ -0,0 +1,49 @@
+# Prefill-decoding Disaggregation (Experimental)
+
+Prefill-decoding disaggregation is a technique that computes the prefill and decoding phases on separate instances, designed mainly for reducing the inteference between the two phases and better utilizing heterogeneous hardware. For each request, following the prefill phase, the system migrates the generated key-value (KV) cache to the decoding instance and continues the computation. 
+
+We find Llumnix well-suited for implementing P-D disaggregation, because this technique is inherently a special request scheduling policy and fits well in Llumnix's modeling for request scheduling. Specifically, P-D disaggregation can be decomposed into two rules (shown below): (1) a special dispatching rule, i.e., P-instances-only; and (2) a special migration rule, i.e., migrate to D instances after one step. Llumnix provides an implementation of P-D disaggregation following this principle.
+
+<div align=center>
+<img src="./pdd_1.png" align="center" width=80%/>
```

Comment:
Update the figure file name to reflect the content of the image for pdd_1.png and pdd_2.png

Reply:
done